src/helix-ui/styles/components/lists.less

```diff
@@ -26,12 +26,14 @@ hx-dl.hxVertical {
 
   hx-dt {
     #grid.span(12);
+    flex: 0 1 auto;
     font-weight: 700;
     text-align: left;
   }
 
   hx-dd {
     #grid.span(12);
+    flex: 0 1 auto;
     display: block;
```

Comment:
The `display` property can be removed, because the columnar flexbox layout will handle the geometry.